### PR TITLE
[FEATURE] Styling store list item components

### PR DESCRIPTION
**Pull request by astr0-4**
* adjusts map and store list container

### CODE REVIEW

**Review by ghost — APPROVED** [review]
LGTM!

File: src/components/StoresList/StoresList.js [hunk 1]
```diff
@@ -10,8 +10,7 @@ const StoresList = props => {
         ? props.storeList.map(store =>
             <li key={store.id}>
               <p>
-                <strong>{store.name}</strong>
-                <br />
+                <h2>{store.name}</h2>
```
Comment (jprittie) [inline src/components/StoresList/StoresList.js]: Hello... I am getting a console error here. It says "Warning: validateDOMNesting(...): 'h2' cannot appear as a descendant of 'p'.

**Review by jprittie — CHANGES_REQUESTED** [review]

File: src/components/StoresList/StoresList.js (hunk 1 above)
Comment (astr0-4) [reply src/components/StoresList/StoresList.js]: good catch @jprittie! I've updated it now.

**Comment by Prtfw** [comment]
👍  LGTM Alex! great css organization

**Review by Prtfw — APPROVED** [review]

**Pull request merged**